### ISSUE TITLE
Test Result

### ISSUE THREAD

**Issue by glee1228** [glee1228/netVLad_Sejong#2]
parameter& Dataset : 
* Dataset : Sejong University building( in Seoul ) Image Dataset
* input image size : ( 128,128,3 )
* base model : VGG16( pretrained weights : 'imagenet')
* netVLAD clustering num : 7
* train batch size : 32
* training triplet data size : 700 ( 100 * 3 pairs images per class )
* total epochs  : 10 
* optimizer : SGD(learning rate=0.01, decay=1e-6, momentum=0.9, nesterov=True)
* triplet loss margin : 1e-4

## Triplet Input Set
![581952303 27](https://user-images.githubusercontent.com/26589942/59275784-f3d0ab80-8c97-11e9-8614-06adf0e347ee.png)

## Input Query Image
![581952216 55](https://user-images.githubusercontent.com/26589942/59275796-f8955f80-8c97-11e9-9d56-166f7564b415.png)

## Top 5 Similar Image in reference Dataset
![581952227 80](https://user-images.githubusercontent.com/26589942/59275806-faf7b980-8c97-11e9-9088-824826230e0f.png)

## TSNE Result
![581952195 79](https://user-images.githubusercontent.com/26589942/59275814-fd5a1380-8c97-11e9-9da4-e95bddfcf9df.png)

**Comment by glee1228** [comment]
# Additional result

<img width="567" alt="TestImage" src="https://user-images.githubusercontent.com/26589942/59279004-1ebdfe00-8c9e-11e9-84fe-07910c06f499.png">
<img width="1070" alt="ResultImage" src="https://user-images.githubusercontent.com/26589942/59279009-2087c180-8c9e-11e9-9636-0964765bc13c.png">
